### PR TITLE
Implementation of getter & setter for accessing MongoDB driver.

### PR DESCRIPTION
**Pull request by mrusme**
@vdemedes @johannesboyne I re-worked what I did in #121 and I'd like to drop the previous PR and use this one instead.

@vdemedes the reason for this refactoring is that not only getting the MongoDB driver is important, but also setting it. In my use case, I would like to use @Opbeat for performance monitoring. Opbeat however expects that it gets the opportunity to "wrap" itself around the MongoDB driver. In order to do so, I'm required to initialise the MongoDB driver inside my project, have Opbeat wrap it and only then pass it through to Mongorito.

The getter on the other hand was discussed inside #121 and should be continued here. Thanks!

### CODE REVIEW

**Comment by vdemedes** [comment]
Could you add a simple test for this change?

**Comment by mrusme** [comment]
@vdemedes please verify. :-)

File: test/attributes.js [hunk 1]
```diff
@@ -36,6 +36,35 @@ test('expose mongodb properties', t => {
 	});
 });
 
+test('set and get mongodb driver', t => {
+	let mockDriver = {
```
Comment (vdemedes) [inline test/attributes.js]: should be `const`.

File: test/attributes.js [hunk 2]
```diff
@@ -36,6 +36,35 @@ test('expose mongodb properties', t => {
...
+		'MongoClient': true
+	};
+
+	mongorito.setDriver(mockDriver);
+
+	t.is(mongorito.getDriver(), mockDriver);
+});
+
+test('set mock mongodb driver and connect', t => {
+	let mockDriver = {
```
Comment (vdemedes) [inline test/attributes.js]: should be `const`.

File: test/attributes.js [hunk 3]
```diff
@@ -36,6 +36,35 @@ test('expose mongodb properties', t => {
...
+		'MongoClient': {
+			'connect': () => {
+				return new Promise(fulfill => {
```
Comment (vdemedes) [inline test/attributes.js]: can just be `return Promise.resolve('ok')`

**Comment by vdemedes** [comment]
Added a few tips, otherwise LGTM.

**Comment by mrusme** [comment]
@vdemedes thanks & adjusted.

**Comment by vdemedes** [comment]
Thanks!

**Pull request merged**